pkg/asset/cluster/cluster.go

```diff
@@ -87,11 +87,13 @@ func (c *Cluster) Generate(parents asset.Parents) (err error) {
 
 	switch {
 	case installConfig.Config.Platform.AWS != nil:
+		identifier := map[string]string{
+			"tectonicClusterID": installConfig.Config.ClusterID,
+		}
+		identifier[fmt.Sprintf("kubernetes.io/cluster/%s", installConfig.Config.ObjectMeta.Name)] = "owned"
 		metadata.ClusterPlatformMetadata.AWS = &types.ClusterAWSPlatformMetadata{
 			Region: installConfig.Config.Platform.AWS.Region,
-			Identifier: map[string]string{
-				"tectonicClusterID": installConfig.Config.ClusterID,
-			},
+			Identifier: identifier,
```

Comment:
do you want to change the Identifier to []map[string]string to more accurately mimic the OR?

Reply:
> do you want to change the Identifier to `[]map[string]string`...

Done in f69e3fd -> 1412168.